app.js

```diff
@@ -0,0 +1,99 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const session = require('express-session');
+const passport = require('passport');
+const { ApolloServer } = require('apollo-server-express');
+const { Op } = require('sequelize');
+
+const db = require('./models');
+const { schema } = require('./graphql');
+
+function application({ authStrategy, graphqlPath, sessionSecret }) {
```

Comment:
does it make sense to pass the session secret as a parameter to this function, or would it be better to specify it as an .env variable?

Reply:
and also, how come we want to pass the `graphqlPath` as a parameter?

Reply:
i think the idea is to do all our env var reading in `server.js` and pass it in to app - honestly no preference one way or another

`graphqlPath` is being used as the route to apply the express middleware on - see line 94